charts/argo-workflows/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.16.6
-appVersion: v3.3.8
+version: 0.16.7
+appVersion: v3.3.8 # TODO: switch to 3.3.9 when available
```

Comment:
do you plan to wait for 3.3.9 or do you plan to keep this TODO in here?  I [queried argo-workflows maintainers](https://cloud-native.slack.com/archives/C020XM04CUW/p1656525154019479?thread_ts=1656525051.021529&cid=C020XM04CUW) and they said 3.3.9 release would be next month sometime, may be a bit.

Reply:
May want to put this into draft mode then, wdyt?

Reply:
Also, that kind of comment would be great in the PR description.

Reply:
I made sure this does not change anything by default outside of the version of the doc, but I'll switch it to a draft, you're right.

Reply:
I missed the linked PR.  You're good. 😄